src/engraving/style/styledef.cpp

```diff
@@ -993,13 +993,13 @@ const std::array<StyleDef::StyleValue, size_t(Sid::STYLES)> StyleDef::styleValue
     { Sid::staffTextFrameBgColor,         "staffFrameBgColor",            PropertyValue::fromValue(draw::Color::transparent) },
 
     { Sid::rehearsalMarkFontFace,         "rehearsalMarkFontFace",        "Edwin" },
-    { Sid::rehearsalMarkFontSize,         "rehearsalMarkFontSize",        14.0 },
+    { Sid::rehearsalMarkFontSize,         "rehearsalMarkFontSize",        8.0 },
```

Comment:
I don't think rehersal marks should that small

Reply:
> @alexpavlov96 Please don't change the style defaults without at least having an argument with me about it first ;)

@oktophonie Sure! I was basically searching what's wrong with the text being so huge and first time it brought me to styles, so I wanted to create a request to ask people tomorrow.